src/api/index.js

```diff
@@ -153,13 +166,37 @@ export async function fetchUser(user, accessToken) {
 }
 
 export async function fetchUserOrgs(user, accessToken) {
-  const ORGS_ENDPOINT = `${root}/users/${user}/orgs`;
+  const getOrganizationsQuery = `
```

Comment:
I'm a bit scared of seeing `api/index.js` grow too much. Would it be possible to put queries in entities subfolders ? This one for example would go in `user/user.queries.js`

Reply:
I agree :+1:

Reply:
_Highly agree_, I made the mistake of laying things all down in `api/index.js` but it really makes sense to leverage feature api files (like we do with other parts of the app).

Reply:
What do you think about moving this query into something like

`/src/queries/foo.query.js` or something like that?

Reply:
Maybe it's better this way `/src/api/queries/foo.query.js`?

Reply:
What about /src/user/user.queries.js ? (just like we'll have user.api.js) ?